LibreNMS/OS/Dlinkap.php

```diff
@@ -52,11 +53,29 @@ public function discoverProcessors()
             Processor::discover(
                 'dlinkap-cpu',
                 $this->getDeviceId(),
-                $this->getDeviceArray()['sysObjectID'] . '.5.1.3.0',  // different OID for each model
+                $this->getDevice()->sysObjectID . '.5.1.3.0',  // different OID for each model
                 0,
                 'Processor',
                 100
             ),
         ];
     }
+
+    public function discoverMempools()
```

Comment:
This OS, and many others, are missing `implements MempoolsDiscovery`

Reply:
I should add it explicitly, the problem is the base OS implements MempoolsDiscovery